src/tosam.py

```diff
@@ -1,18 +1,21 @@
-import argparse
-
-
-def main():
-    argparser = argparse.ArgumentParser(description="To Simple-SAM converter")
-    argparser.add_argument(
-        "mas",
-        type=argparse.FileType('r')
-    )
-    args = argparser.parse_args()
-    print(args)
-    for line in args.mas:
-        chrom, read_name, read_str, pos = line.split('\t')
-        # Output as Simple-SAM
-
-
-if __name__ == '__main__':
-    main()
+import argparse
+import sys
+
+parser = argparse.ArgumentParser(prog = "tosam")
+parser.add_argument("-path", nargs = 1, type = str)
+
+args = parser.parse_args()
+
+file_object = open(r"{path}".format(path=args.path[0]), "r")
+
+listy = list()
+for line in file_object:
+	listy.append(line.strip().split("\t"))
+
+for i in listy:

```

Comment:
A cleaner version would be:

```python
for refname, readname, readstring, position in listy:
    ...
```

where you get Python to do the unpacking. It will do it much faster, since it will unpack a tuple in C and you are indexing into it multiple times in Python.